LEAF_Request_Portal/templates/view_reports.tpl

```diff
@@ -281,17 +281,19 @@ function loadSearchPrereqs() {
                 var i = groupNames[k].categoryID;
                 var associatedCategories = groupIDmap[i].categoryID;
                 if(groupIDmap[i].parentCategoryID != '') {
-                    associatedCategories += ' ' + groupIDmap[i].parentCategoryID; 
+                    associatedCategories += ' ' + groupIDmap[i].parentCategoryID;
                 }
                 if(groupIDmap[i].parentStaples != null) {
                     for(var j in groupIDmap[i].parentStaples) {
                         associatedCategories += ' ' + groupIDmap[i].parentStaples[j];
                     }
                 }
-                
+
                 var categoryLabel = groupNames[k].categoryName;
                 if(groupIDmap[i].parentCategoryID != '') {
-                    categoryLabel += "<br />" + groupIDmap[groupIDmap[i].parentCategoryID].categoryName;
+                    if(groupIDmap[groupIDmap[i].parentCategoryID]){
```

Comment:
Use `&&` instead of nesting if statements here.

Reply:
done